common/solana_rpc/alt_builder.py

```diff
@@ -109,46 +110,26 @@ async def rebuild_to_real_alt(self, alt: SolAltInfo) -> SolAltInfo:
     def can_merge_alt(dst_alt: SolAltInfo, src_alt: SolAltInfo) -> bool:
         return len(dst_alt.account_key_list) + len(src_alt.account_key_list) < SolAltProg.MaxAltAccountCnt
 
-    def build_alt_tx_set(self, alt: SolAltInfo) -> SolAltTxSet:
-        is_alt_exist = alt.is_exist
-
-        # Tx to create an Address Lookup Table
-        create_alt_tx_list: list[SolLegacyTx] = list()
-        if not is_alt_exist:
-            ix_list = tuple(
-                [
-                    self._cb_prog.make_cu_price_ix(self._cu_price),
-                    self._cb_prog.make_cu_limit_ix(self._alt_prog.CuLimitCreate),
-                    self._alt_prog.make_create_alt_ix(alt.ident),
-                ]
-            )
-            create_alt_tx = SolLegacyTx(name=self._create_name, ix_list=ix_list)
-            create_alt_tx_list.append(create_alt_tx)
-
+    def build_ext_alt_tx_set(self, alt: SolAltInfo) -> SolAltTxSet:
```

Comment:
What is the purpose to add `_ext_` in the name?
Should external modules know about the internal implementation?